.github/workflows/ci-workflow.yml

```diff
@@ -37,6 +37,13 @@ jobs:
           TAG_ID=${GITHUB_REF##*/}
           echo "ACTIONS_TAG=$TAG_ID" >> $GITHUB_ENV
 
+      - name: SonarCloud Scan
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: |
+          ./gradlew sonarqube -Dsonar.login=$SONAR_TOKEN --info
```

Comment:
This should be in its own workflow and run on pull requests and long lived branch (`main`, `develop`, `support-*`) commits. See mgmt-api's workflow for an example: https://github.com/Alvearie/hri-mgmt-api/blob/develop/.github/workflows/code-scans.yml